src/web/components/ClientSideTokenGeneration/CstgDomainsTable.stories.tsx

```diff
@@ -25,9 +25,7 @@ export const WithoutDomains: Story = {
     domains: [],
     onUpdateDomains: (domains: string[]) => {
       Promise.resolve(console.log('update domains to:', domains));
```

Comment:
Not your code, but why is this console.log() in a Promise.resolve()?  Seems weird.

Reply:
I think this is just our pattern to console log inputs in storybooks where the prop returns a promise. Is there a better way to do that?